### ISSUE TITLE
Ghostwriter do not detect pandoc on MacOS 10.14.6

### ISSUE THREAD

**Issue by Korsani** [wereturtle/ghostwriter#466]
Although pandoc is installed via brew, Ghostwriter do not offer me pandoc export

```
~/ghostwriter$ which pandoc
/usr/local/bin/pandoc
```

Ghostwriter 1.8.0

**Comment by franz-sp** [comment]
It used to be broken also on ghostwriter v2.0.0-rc, just re-compiled today.
But I noticed that if I don't start ghostwriter clicking the app icon but by typing `ghostwriter` in the command-line (after `$ sudo ln -s /Applications/ghostwriter.app/Contents/MacOS/ghostwriter /usr/local/bin` as suggested in the ReadMe) Pandoc appears.
I also noticed that clicks on the icons on the lower part of the main window (the one that contains the "Copy to HTML" and "Hemingway mode", to be clear) are irresponsive if I start the application from the app icon, but work when I start it from the command line.

MacOs 10.15.6 on MacBookPro mid 2012.
Hope it helps

F
